CHANGELOG/v1.12.1.md

```diff
@@ -0,0 +1,27 @@
+## Changes by Kind
+
+### Bug or Regression
+
+- Fix: DNSPrefix error on existing tenant cluster (#4368, @mihaiandreiratoiu)
```

Comment:
Is it important that the plain references like #4368 don't seem to get auto-linked here in the rendered version of this doc like they presumably will be on the release page?

Reply:
turns out there is a way to specify an option for full links, good call! https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md

will add